qml/components-Sailfish/IconButton.qml

```diff
@@ -0,0 +1,30 @@
+import Sailfish.Silica 1.0
+import "." as BVApp
+
+IconButton {
+    function iconBy(type) {
```

Comment:
They have something how the icon should look like AFTER it was pressed (what we actually need for the upcoming favourites feature), but not for the pressed state itself.

I am not perfectly clear, what you want to achieve. So I'd suggest to move the function as it is, so it is at the right place and to postpone further improvements. I have to think about the whole VenueDetails-design anyways.

Ok?